docs/Onboarding.md

```diff
@@ -60,6 +60,9 @@ export DB_DEFAULT="{
 poetry run python3 manage.py runserver
 ```
 
+> Note:
+> It is possible to have authentication issues escaping special characters. In some cases, it is necessary to add more than one backslash, while in others, no addition is needed. One way to see how your computer is handling the escapes is to add a print statement for the environment variable `DB_DEFAULT` in the `manage.py` file with `print(os.getenv('DB_DEFAULT'))`.
```

Comment:
I think a more realistic approach would be to print the `DATABASES` variable  in

``` 
DATABASES = {
    "default": get_json_env_var(
        "DB_DEFAULT",
        {
            "ENGINE": "django.db.backends.postgresql",
            "NAME": "kernelci",
            "USER": "kernelci",
            "PASSWORD": "kernelci-db-password",
            "HOST": "127.0.0.1",
            "OPTIONS": {
                "connect_timeout": 5,
            },
        },
    )
}
```

Since we are having much trouble with escaped characters, it is a good idea to print `if DEBUG` is equal to True

Reply:
Also, leave that print as a bug for extra precaution, I don't think it has any danger of printing passwords in production, but better safe than sorry